imports/ui/pages/EditDocument.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-import DocumentEditor from '../components/DocumentEditor.js';
+import DocumentEditor from '../components/DocumentEditor';
+import NotFound from './NotFound';
 
-const EditDocument = ({ doc }) => (
-  <div className="EditDocument">
-    <h4 className="page-header">Editing "{ doc.title }"</h4>
-    <DocumentEditor doc={ doc } />
-  </div>
-);
+const EditDocument = ({ doc }) => {
```

Comment:
One little tweak, just to be concise:

```javascript
const EditDocument = ({ doc }) => {
  return doc ? (
    <div className="EditDocument">
      <h4 className="page-header">Editing "{ doc.title }"</h4>
      <DocumentEditor doc={ doc }/>
    </div>
  ) : <NotFound />;
};
```

We can use a ternary here to condense this down :)